src/components/pages/ParentFamily/ParentFamilyHome.js

```diff
@@ -2,110 +2,120 @@ import React, { useState, useEffect } from 'react';
 import axios from 'axios';
 import Banner from '../../common/Banner';
 import ParentSidebar from '../ParentHome/ParentSidebar';
-// import CreateNewStudent from './CreateNewStudent';
-// import { useParams } from 'react-router';
-// import CurrentCoursesDetails from './CurrentCoursesDetails';
+import CreateNewStudent from './CreateNewStudent';
 import '../../../styles/ParentStyles/index.less';
+import { Layout, Modal, Button, Card, Avatar, Col, Row } from 'antd';
+import 'antd/dist/antd.css';
 
 const ParentFamilyHome = () => {
-  const [studentInfo, setStudentInfo] = useState(null); //eslint-disable-line
-  // const [modal, setModal] = useState(false);
-  // const [modal2, setModal2] = useState(false);
+  const { Meta } = Card;
+  const { Content } = Layout;
+  const [studentInfo, setStudentInfo] = useState(null);
+  const [addStudentVisible, setAddStudentVisible] = useState(false);
+  const [addStudentConfirmLoading, setAddStudentConfirmLoading] = useState(
+    false
+  );
 
   useEffect(() => {
     const token = JSON.parse(localStorage.getItem(`okta-token-storage`));
     const config = {
       headers: { Authorization: `Bearer ${token.idToken.value}` },
     };
     axios
-      .get(`https://coder-heroes-api.herokuapp.com/parent/1/children`, config)
+      .get(`https://coder-heroes-api.herokuapp.com/parent/1/Studentren`, config)
       .then(res => {
         const familyData = res.data;
         setStudentInfo(familyData);
+        console.log(familyData);
       })
       .catch(err => {
         console.log(`error fetching axios call`);
       });
   }, []);
 
+  const showAddStudentModal = () => {
+    setAddStudentVisible(true);
+  };
+
+  const handleAddStudentOk = () => {
+    setAddStudentConfirmLoading(true);
+    setTimeout(() => {
+      setAddStudentVisible(false);
+      setAddStudentConfirmLoading(false);
+    }, 2000);
+  };
+
+  const handleAddStudentCancel = () => {
+    setAddStudentVisible(false);
+  };
+
   return (
-    <div className="family-page-container">
+    <Layout style={{ width: '100%' }}>
```

Comment:
If possible, I would try to keep all of teh styling in a separate file instead of doing in straight in html.